subiquitycore/utils.py

```diff
@@ -35,21 +35,24 @@ def _clean_env(env, *, locale=True):
     return env
 
 
-def orig_environ(env):
+def orig_environ(env: Optional[Dict[str, str]]) -> Dict[str, str]:
     """Generate an environment dict that is suitable for use for running
     programs that live outside the snap."""
+
     if env is None:
-        env = os.environ
-    ret = env.copy()
+        env: Dict[str, str] = os.environ
+
+    ret: Dict[str, str] = env.copy()
+
     for key, val in env.items():
         if key.endswith("_ORIG"):
-            key_to_restore = key[: -len("_ORIG")]
+            key_to_restore: str = key[: -len("_ORIG")]
             if val:
                 ret[key_to_restore] = val
             else:
                 del ret[key_to_restore]
             del ret[key]
-    ret.pop("LD_LIBRARY_PATH", None)
+    _ = ret.pop("LD_LIBRARY_PATH", None)
```

Comment:
The convention we're using in Subiquity is to not assign to underscore, as it's used for translation strings.  Do we need to assign to something?

Reply:
Nope, this was just a personal preference when I was making edits.If that's convention I'll remove it.